pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -34,16 +35,21 @@ import (
 )
 
 const (
-	pipelinesSCCClusterRole  = "pipelines-scc-clusterrole"
-	pipelinesSCCRoleBinding  = "pipelines-scc-rolebinding"
-	pipelinesSCC             = "pipelines-scc"
-	pipelineSA               = "pipeline"
+	pipelinesSCCClusterRole = "pipelines-scc-clusterrole"
+	pipelinesSCCRoleBinding = "pipelines-scc-rolebinding"
+	pipelinesSCC            = "pipelines-scc"
+	pipelineSA              = "pipeline"
+	PipelineRoleBinding     = "openshift-pipelines-edit"
+	// TODO: Remove this after 1.7.x releases
```

Comment:
Note: "1.7.x releases" is a downstream "thing", it doesn't have any meaning here (upstream). We should most likely say "TODO: Remove this after a deprecation notice time, which would be 3 weeks" or something..

Reply:
i agree :)